Containerfile

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: MIT
+#
+# Copyright (C) 2024 Olliver Schinagl <oliver@schinagl.nl>
+
+ARG UBUNTU_VERSION="22.04"
+ARG TARGET_ARCH="library"
+
+FROM index.docker.io/${TARGET_ARCH}/ubuntu:${UBUNTU_VERSION}
+
+ENV HOME="/build"
+
+ARG SLF2_COMPONENTS="PF_CC2X"
+ENV SLF2_COMPONENTS="${SLF2_COMPONENTS}"
+
+ARG CCS_VERSION="12.5.0"
```

Comment:
You might need to bump the version, bacause the patch expect Sysconfig 18.1+ but the version 18.0 is delivered in this version of CCS.
```
Error: SimpleLink Low Power F2 SDK version 7.41.00.17 requires at least version 1.18.1 of SysConfig.
```